postProcess/remove_spikes.m

```diff
@@ -0,0 +1,52 @@
+function rez = remove_spikes(rez,remove_idx,label,varargin)
```

Comment:
this function encapsulates the code for removing spikes from a "rez" structure, so that it can easily be shared by set_cutoff.m and remove_ks2_duplicated_spikes.m. It also adds a field to the rez structure called "removed" which saves the removed spikes and provides a reason why they were removed in postprocessing.